### PR TITLE
Remove right sidebar on home page and implement design changes 

### PR DESCRIPTION
**Pull request by annielovesjs**
Changes:
- remove right side bar from home page
- changed styling on the front page

How to test:
- should look like this:
homepage

### CODE REVIEW

**Review by piotrmark — DISMISSED** [review]

File: src/client/src/translations/en.json [hunk 1]
```diff
@@ -122,9 +122,7 @@
   "privacyModal.privacyStatement": "Privacy Statement",
   "privacyModal.thirdPartyWebsite": "You will be taken to a third-party website which is a non-Microsoft service.",
   "projectName.ariaProjectName": "Project Name Input",
-  "projectName.outputPath": "Output Path",
   "projectName.outputPathTitle": "Output Path:",
-  "projectName.projectName": "Project Name",
   "projectName.projectNameTitle": "Project Name:",
```
Comment (jasmingoh) [inline src/client/src/translations/en.json]: Is this still required? seems like it is removed in the whitelist

File: src/client/src/translations/en.json [hunk 2]
```diff
@@ -122,9 +122,7 @@
   "privacyModal.privacyStatement": "Privacy Statement",
   "privacyModal.thirdPartyWebsite": "You will be taken to a third-party website which is a non-Microsoft service.",
   "projectName.ariaProjectName": "Project Name Input",
-  "projectName.outputPath": "Output Path",
   "projectName.outputPathTitle": "Output Path:",
```
Comment (jasmingoh) [inline src/client/src/translations/en.json]: Is this still required? seems like it is removed in the whitelist

File: src/client/src/translations/whitelist_en.json [hunk 3]
```diff
@@ -96,11 +96,7 @@
   "privacyModal.beingRedirected",
   "privacyModal.privacyStatement",
   "privacyModal.OK",
-  "projectName.projectNameTitle",
-  "projectName.projectName",
   "projectName.ariaProjectName",
-  "projectName.outputPathTitle",
```
Comment (jasmingoh) [inline src/client/src/translations/whitelist_en.json]: seems like this is still being used..

File: src/client/src/translations/whitelist_en.json [hunk 4]
```diff
@@ -96,11 +96,7 @@
   "privacyModal.beingRedirected",
   "privacyModal.privacyStatement",
   "privacyModal.OK",
-  "projectName.projectNameTitle",
```
Comment (jasmingoh) [inline src/client/src/translations/whitelist_en.json]: seems like this is still being used too

**Comment by jasmingoh** [comment]
@annielovesjs did you run `npm manage:translations`  in src/client?

File: src/client/src/translations/whitelist_en.json (hunk 4 above)
Comment (streetcoder123) [reply src/client/src/translations/whitelist_en.json]: These are automatically generated when we build. @annielovesjs you should probably revert them to original if these changes were not intentionally made.

File: src/client/src/translations/whitelist_en.json (hunk 3 above)
Comment (annielovesjs) [reply src/client/src/translations/whitelist_en.json]: Sorry, this was automatically removed - just added it back for both

File: src/client/src/translations/whitelist_en.json (hunk 4 above)
Comment (annielovesjs) [reply src/client/src/translations/whitelist_en.json]: Yup! Just reverted it

**Comment by annielovesjs** [comment]
> @annielovesjs did you run `npm manage:translations` in src/client?

I normally just change the text directly in the translations folder but that's definitely a better approach that I'll use from now on

**Review by streetcoder123 — APPROVED** [review]
Looks way better :)

**Review by jasmingoh — APPROVED** [review]

**Review by piotrmark — APPROVED** [review]

**Review by jcoc611-microsoft — APPROVED** [review]
Nice!

**Comment by SahilTara** [comment]
Something that you should look at maybe in another PR is the centering it feels a bit off might just be me though

**Pull request merged**